pkg/jx/cmd/create_cluster_gke.go

```diff
@@ -114,6 +123,8 @@ func NewCmdCreateClusterGKE(commonOpts *opts.CommonOptions) *cobra.Command {
 
 	options.addCreateClusterFlags(cmd)
 
+	cmd.Flags().StringVarP(&options.Flags.CloudBeesDomain, "cloudbees-domain", "", "", "When setting up a letter/tenant cluster, this creates a tenant cluster on the cloudbees domain which is retrieved via the required URL")
+	cmd.Flags().StringVarP(&options.Flags.CloudBeesAuth, "cloudbees-auth", "", "", "Auth used when setting up a letter/tenant cluster")
```

Comment:
let's address this comment in a followup PR but maybe we need to describe the format that the auth string needs to be provided as in the description.

Reply:
totally, I missed that